react/features/welcome/components/WelcomePage.web.js

```diff
@@ -198,6 +198,7 @@ class WelcomePage extends AbstractWelcomePage {
                                 { t('welcomepage.enterRoomTitle') }
                             </div>
                             <form onSubmit = { this._onFormSubmit }>
+                                <label htmlFor = 'enter_room_field'>Room name:</label>
```

Comment:
This should be hopefully localizable.

See: https://github.com/jitsi/jitsi-meet/blob/82b9cb77447a74ef6812b35b9e46ec1971b0e5ce/react/features/welcome/components/WelcomePage.web.js#L188

Also: https://github.com/jitsi/jitsi-meet/tree/3f679407a93d033a2a99824d06bfe806eae8e077/lang#development

> If you want to add new functionality for Jitsi Meet and you have texts that need to be translated please use our translation module. It is located in modules/translation. You must add key and value in main.json file in English for each translatable text. Than you can use the key to get the translated text for the current language.
> 
> WARNING: Please don't change the other language files except main.json! They must be updated and translated via our Pootle user interface!

Reply:
Thanks for the hint! I added a label and a translation accordingly.

Reply:
Hi @luixxiul ! Would it be possible for you to re-review the change? I added the translation as you proposed.
Thanks!

Reply:
Thanks for the ping!

While I do not have the authority to merge the PR as I'm just a contributor, I am going to check the PR to see if the change should work as expected 😄